app/src/main/java/org/android/go/sopt/di/RetrofitModule.kt

```diff
@@ -0,0 +1,64 @@
+package org.android.go.sopt.di
+
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.serialization.json.Json
+import okhttp3.Interceptor
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.Response
+import okhttp3.logging.HttpLoggingInterceptor
+import org.android.go.sopt.BuildConfig
+import retrofit2.Retrofit
+import java.io.IOException
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object RetrofitModule {
+    @Singleton
+    @Provides
+    fun provideOkHttpClient() = if (BuildConfig.DEBUG) {
+        val loggingInterceptor = HttpLoggingInterceptor()
+        loggingInterceptor.setLevel(HttpLoggingInterceptor.Level.BODY)
+        OkHttpClient.Builder()
+            .addInterceptor(loggingInterceptor)
+            .build()
+    } else {
+        OkHttpClient.Builder().build()
+    }
+
+
+    private const val USER_BASE_URL = BuildConfig.USER_BASE_URL
+    private const val REQRES_BASE_URL = BuildConfig.REQRES_BASE_URL
+    @Singleton
+    @Provides
+    @ApiModule.SignRetrofit
+    fun provideSpringRetrofit(okHttpClient: OkHttpClient): Retrofit = Retrofit.Builder()
+        .baseUrl(USER_BASE_URL)
+        .client(okHttpClient)
+        .addConverterFactory(Json.asConverterFactory("application/json".toMediaType()))
+        .build()
+
+    @Singleton
+    @Provides
+    @ApiModule.ReqresRetrofit
+    fun provideReqresRetrofit(okHttpClient: OkHttpClient): Retrofit = Retrofit.Builder()
+        .baseUrl(REQRES_BASE_URL)
```

Comment:
```suggestion
    import org.android.go.sopt.BuildConfig.USER_BASE_URL
    import org.android.go.sopt.BuildConfig.REQRES_BASE_URL
    
    @Singleton
    @Provides
    @ApiModule.SignRetrofit
    fun provideSpringRetrofit(okHttpClient: OkHttpClient): Retrofit = Retrofit.Builder()
        .baseUrl(USER_BASE_URL)
        .client(okHttpClient)
        .addConverterFactory(Json.asConverterFactory("application/json".toMediaType()))
        .build()

    @Singleton
    @Provides
    @ApiModule.ReqresRetrofit
    fun provideReqresRetrofit(okHttpClient: OkHttpClient): Retrofit = Retrofit.Builder()
        .baseUrl(REQRES_BASE_URL)
```
상수를 import 해주면 변수를 따로 선언할 필요가 없어질 것 같습니다!